packages/rule-x-content-type-options/README.md

```diff
@@ -1,35 +1,33 @@
 # Require `X-Content-Type-Options` HTTP response header (`@sonarwhal/rule-x-content-type-options`)
 
-`x-content-type-options` warns against not serving scripts and
-stylesheets with the `X-Content-Type-Options: nosniff` HTTP response
-header.
+`x-content-type-options` checks that all scripts and
+stylesheets are sent to the browser with the
+`X-Content-Type-Options: nosniff` HTTP response header.
```

Comment:
Let' use: `requires that all scripts and stylesheets are served with the X-Content-Type-Options: nosniff
HTTP response header.`.